cmd/juno/cli/root.go

```diff
@@ -30,41 +36,80 @@ var (
 	//go:embed long.txt
 	longMsg string
 
+	processHandler *process.Handler
 	// rootCmd is the root command of the application.
 	rootCmd = &cobra.Command{
 		Use:   "juno [options]",
 		Short: "Starknet client implementation in Go.",
-		Long:  longMsg,
-		Run: func(cmd *cobra.Command, args []string) {
-			handler := process.NewHandler()
+		Run: func(_ *cobra.Command, _ []string) {
+			processHandler = process.NewHandler()
 
 			// Handle signal interrupts and exits.
 			sig := make(chan os.Signal)
 			signal.Notify(sig, os.Interrupt, syscall.SIGTERM)
 			go func() {
 				<-sig
```

Comment:
Passing the `sig` channel as a goroutine parameter, it's good practice to access external context variables with parameters and not directly